app/src/views/private/components/render-template.vue

```diff
@@ -153,6 +153,7 @@ function handleObject(fieldKey: string) {
 @import '@/styles/mixins/no-wrap';
 
 .render-template {
+	height: 100%;
```

Comment:
I'm pretty sure there was a reason it was removed from the global render-template in the first place! Lets make sure this doesn't introduce any nasty side effects elsewhere 👍🏻

Reply:
Rijk is correct here as this PR coincidentally is a direct revert of #14231! 😄

Technically re-adding `height: 100%` does re-introduce the previous bug #13010, however it also seems like the fix in #15115 also resolves that bug since now it overrides `height: 100%` with `height: v-bind('cssHeight.tableRow')` from the `<table-row>` level:

https://user-images.githubusercontent.com/42867097/202343711-11381ebe-c091-4eec-b92f-dd9e828c0c8e.mp4

Reply:
So with this changes it works fine in all cases?

To avoid setting `height: 100%` globally, we can achieve the same result with adding it locally inside `one-to-many` and `v-list` interfaces.